### DISCUSSION TITLE
Add Volume to Plugin API

### DISCUSSION THREAD

**Discussion by jacob-willden** [jellyfin/jellyfin#9707] [General] [answered]
I've been developing a plugin for Jellyfin that will allow for the use of edit decision list (EDL) files, which can be used for cutting out offensive, boring, tedious, or otherwise undesirable content in one's videos. This is desirable over just cutting the videos in a video editing program because it's actually allowed under the [United States Family Entertainment Act of 2005](https://www.congress.gov/bill/109th-congress/house-bill/357/text). 

Unfortunately, Jellyfin's plugin API does not yet have a way to programmatically adjust the volume or mute/un-mute a video. Adding this would be beneficial for [any users who want EDL functionality](https://features.jellyfin.org/posts/180/support-for-edl-comskip-files), as well as any plugins that could use the ability to adjust the volume of a video. It would also help bring over users who are sticking with Kodi because they rely on [Kodi's EDL functionality](https://kodi.wiki/view/Edit_decision_list), and could be attractive to Plex [users who are frustrated with the lack of EDL functionality that Plex provides](https://www.reddit.com/r/PleX/comments/golv1t/manual_skip_tv_show_intros/).

**Comment by nielsvanvelzen** [comment, marked as answer]
Relevant: https://github.com/jellyfin/jellyfin-meta/discussions/30
